### ISSUE TITLE
[BUG] Test forum : User can unhide message in self.

### ISSUE THREAD

**Issue by luluwebmaster** [epochtalk/epochtalk#505]
Hello,

When you open a thread in self moderated mode and you hide a message of another user he still can unhide it himself.

Original bug post : https://www.cryptos-currencies.com/threads/fd5dMTGLSJilQPxsZtmhmg/posts?start=9#eWjLZv0mSuiZdr0nSB0NAQ

**Comment by akinsey** [comment]
@luluwebmaster this should now be fixed, you will have to reload the base permissions set for this to take effect though. Currently there is no clean way of doing this, you have to wipe the  base roles from the roles table. We will find a more elegant solution in the future, but for now:

1. Make sure you're on the latest of `fixes-2019` for epochtalk
2. Connect to your postgres database and run the following sql commands:
```
SET session_replication_role = 'replica';
DELETE FROM roles WHERE lookup IN ('moderator', 'anonymous', 'globalModerator', 'newbie', 'banned', 'superAdministrator', 'patroller', 'private', 'user', 'administrator');
SET session_replication_role = 'origin';
```
3. Restart the epochtalk server, this will reload the roles with the new base permissions

After doing so, self mod features should work as expected. Just let me know if you have any issues

**Comment by unenglishable** [comment]
resolved by #534
